client-src/index.js

```diff
@@ -83,6 +91,23 @@ if (parsedResourceQuery.overlay) {
       runtimeErrors: true,
       ...options.overlay,
     };
+
+    ["errors", "warnings", "runtimeErrors"].forEach((property) => {
+      if (typeof options.overlay[property] === "string") {
```

Comment:
This logic could be a breaking change for those passing `string` to the options, as we will assume string = encoded function, meanwhile previously it was coerce to boolean.

Reply:
Maybe not since we have schema validation.